{{cookiecutter.github_repository}}/docs/backend/continuous_integration.md

```diff
@@ -0,0 +1,102 @@
+# Travis
+
+	TODO
+
+# Circleci
+
+	TODO
+
+# Drone.io and Drone
+
+## Setting up Django Init with Drone.io Continuous Integration
```

Comment:
- Can remove the reference to `Django Init` from here. After the code has been generated it has no relation with django-init.